packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -444,4 +444,60 @@ class ControlledMultiSelectExample extends React.Component {
 }
 
 <ControlledMultiSelectExample />
-```
\ No newline at end of file
+```
+
+### Multi select with search filter
+
+When there are many options, you could use a search filter in the MultiSelect. The search filter will be performed toward the labels of the option items.
+
+```js
+import {MultiSelect, ActionItem, SeparatorItem, OptionItem} from "@khanacademy/wonder-blocks-dropdown";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {StyleSheet} from "aphrodite";
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+    },
+});
+
+const optionItems = new Array(10).fill(null).map((_, i) => (<OptionItem
+    key={i}
+    value={(i + 1).toString()}
+    label={`School ${i + 1} in Wizarding World`}
+/>));
+
+class ExampleWithShortcuts extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            selectedValues: [],
+        };
+        // Styleguidist doesn't support arrow functions in class field properties
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(update) {
+        this.setState({
+           selectedValues: update,
+        });
+    }
+
+    render() {
+        return <MultiSelect
+            shortcuts={true}
+            isFilterableByLabel={true}
```

Comment:
I thought that shortcuts aren't shown when `isFilterableByLabel` is true.  It would be nice to enforce this constraint using flow if possible.